gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/orchestration_stack.rb

```diff
@@ -21,6 +23,14 @@ def raw_update_stack(template, options)
     raise MiqException::MiqOrchestrationUpdateError, err.to_s, err.backtrace
   end
 
+  def self.fix_options(options)
```

Comment:
@bzwei Instead of the generic name `fix_options` maybe this should be `parameter_format_v2`